### PR TITLE
fix: add best iteration on TrainingHome

### PR DESCRIPTION
**Pull request by alx**

### CODE REVIEW

File: src/stores/modelRepositories/repository.js [hunk 1]
```diff
@@ -129,6 +135,16 @@ export default class Repository {
 
       if(['acc', 'accp', 'meaniou', 'map', 'f1'].includes(metricKey)) {
         value = Math.max.apply(Math, measure_hist[metricHistKey])
+      } else if(['iteration'].includes(metricKey)) {
+        const bestTrainLossIndex = measure_hist["train_loss_hist"].indexOf(
```
Comment (Bycob) [inline src/stores/modelRepositories/repository.js:139]: Why are you taking the best loss? The idea here would be to take the value stored in the best_model.txt file

Comment (alx) [reply src/stores/modelRepositories/repository.js]: add `bestModel['iteration']` data fetching if exists in [a9d7e8c](https://github.com/jolibrain/platform_ui/pull/67/commits/a9d7e8cabc96f0aa6e189221f3864536bead7505)

File: src/stores/modelRepositories/repository.js [hunk 2]
```diff
@@ -139,6 +145,30 @@ export default class Repository {
...
+      } else if(['iteration'].includes(metricKey)) {
+
+        if(
+          this.bestModel !== null &&
+            'iteration' in this.bestModel
+        ) {
+
+          value = this.bestModel['iteration']
+
+        } else {
```
[the remark this answers is not in the document]
Comment (alx) [reply src/stores/modelRepositories/repository.js]: thanks, fixed in https://github.com/jolibrain/platform_ui/commit/fd105971e8b80842a049ca92101ce38f877a3e65

**Review by Bycob — APPROVED** [review]

**Comment by Bycob** [comment]
Squash commit before merge

**Pull request merged**